src/Attributes/RequestBody.php

```diff
@@ -12,13 +12,14 @@
 class RequestBody extends \OpenApi\Annotations\RequestBody
 {
     /**
+     * @param array<string, MediaType>|JsonContent|XmlContent|null          $content
      * @param array<string,string>|null $x
      * @param Attachable[]|null         $attachables
      */
     public function __construct(
         ?string $description = null,
         ?bool $required = null,
-        $content = null,
+        ?array $content = null,
```

Comment:
I think this line should be 
```php
        array|JsonContent|XmlContent|null $content = null,
```

Reply:
Union types were only introduced in PHP 8.0, but swagger-php still supports PHP 7.2, so this would not work.

Reply:
Ah, nice, I wouldn't have thought of this possibility!